azure-pipelines.yml

```diff
@@ -420,20 +420,23 @@ stages:
               vs2017_latest89:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "LATEST_RELEASE[8.9]"
+                SONAR_DOTNET_VERSION: '9.7.0.75501' # sonar-dotnet 9.8 will mention 9.9 as minimum supported version
                 SONAR_CFAMILYPLUGIN_VERSION: "6.20.5.49286"  # LATEST_RELEASE of CFAMILY is not compatible with old SQ
                 MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\MSBuild.exe"
                 JDKVERSION: "1.11"
                 TEST_SUITE: "**/SonarQubeTestSuite.java"
               vs2019_latest79:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "LATEST_RELEASE[7.9]"
```

Comment:
I will remove this in https://github.com/SonarSource/sonar-scanner-msbuild/pull/1662